nvflare/private/fed/utils/fed_utils.py

```diff
@@ -415,20 +414,15 @@ def set_scope_prop(scope_name: str, key: str, value: Any):
         value: value of property
     Returns: None
     """
-    check_str("scope_name", scope_name)
-    check_str("key", key)
-    data_bus = DataBus()
-    data_bus.put_data(_scope_prop_key(scope_name, key), value)
+    set_scope_property(scope_name, key, value)
 
 
-def get_scope_prop(scope_name: str, key: str) -> Any:
+def get_scope_prop(scope_name: str, key: str, default=None) -> Any:
     """Get the value of a specified property from the specified scope.
     Args:
         scope_name: name of the scope
         key: key of the scope
+        default: value to return if the prop is not found
     Returns:
     """
-    check_str("scope_name", scope_name)
-    check_str("key", key)
-    data_bus = DataBus()
-    return data_bus.get_data(_scope_prop_key(scope_name, key))
+    return get_scope_property(scope_name, key, default)
```

Comment:
What is the reasoning for having these functions in addition to the ones added in fuel.data_event.utils?

Reply:
Just for backward compatibility - some places are already using fed_utils.

The real reason to move this function to nvflare.fuel.data_event.utils is for cleaner dependency. We don't want stuff in app_common (e.g. those client API stuff) to depend on fed_utils, which is in "private".